documentation/client-side-forms/tutorial-validation-ts.asciidoc

```diff
@@ -16,13 +16,88 @@ For each model, there is an associated `BinderNode`, which you can add validator
 BinderNode name =  binder.for(this.binder.model.name);
```

Comment:
BTW now that you’re here, could you fix the syntax on this line? In TypeScript should be:

```
const binderNode = this.binder.for(this.binder.model.name);
```

Reply:
Done.